### ISSUE TITLE
Unexpected client-side exception thrown when checking if index exists

### ISSUE THREAD

**Issue by shravan2x** [elastic/elasticsearch-net#5734]
**NEST/Elasticsearch.Net version**: 7.13.0

**Elasticsearch version**: 7.12.1

**Description of the problem including expected versus actual behavior**:
An exception is thrown when checking if the index exists. The request seems to succeed (status code 200) but the client throws anyway. This code worked fine till a few days ago.

```
Unhandled exception. Elasticsearch.Net.ElasticsearchClientException: Request failed to execute. Call: Status code 200 from: HEAD /streaming_provider
   at Elasticsearch.Net.Transport`1.HandleElasticsearchClientException(RequestData data, Exception clientException, IElasticsearchResponse response)
   at Elasticsearch.Net.Transport`1.FinalizeResponse[TResponse](RequestData requestData, IRequestPipeline pipeline, List`1 seenExceptions, TResponse response)
   at Elasticsearch.Net.Transport`1.RequestAsync[TResponse](HttpMethod method, String path, CancellationToken cancellationToken, PostData data, IRequestParameters requestParameters)
   at ...
```

**Steps to reproduce**:
 1. `await _elasticClient.Indices.ExistsAsync(indexName);`

**Comment by stevejgordon** [comment]
Hi @shravan2x. Thanks for reporting this. This is a duplicate of #5730 and I'm looking into this. May I ask please how your Elasticsearch cluster is deployed? Is it an Elastic Cloud deployment? If so, could you please let me know which cloud provider and region you are using?

**Comment by stevejgordon** [comment]
@shravan2x - We isolated this to a change in 7.13.0 which affects these requests against Elastic Cloud deployments. I have released 7.13.1 which includes a fix for this issue from #5735.

**Comment by shravan2x** [comment]
You're right, it's an Elastic Cloud deployment on AWS. Thanks for the quick fix!
